index.html

```diff
@@ -2205,10 +2205,26 @@ <h3>Verification Method Binding</h3>
       <section>
         <h3>Verification Relationship Validation</h3>
 
-        <p class="issue">
-Implementers need to ensure that when a verification method is used, that
-it matches the verification relationship associated with it and that it lines
-up with the proof purpose.
+        <p>
+When an implementation is <a href="#verify-proof">verifying a proof</a>, it is
+imperative that they verify not only that the <a>verification method</a> used to
+generate the proof is listed in the <a>controller document</a>, but also that it
+was intended to be used to generate the proof that is being verified. This process
+is known as "verification relationship validation".
+        </p>
+        <p>
+The process for verification relationship validation is outlined in
+Section <a href="#retrieve-cryptographic-material"></a>.
+        </p>
+        <p>
+This process is used to ensure that cryptographic material, such as a private
+cryptographic key, is not misused by application to an unintended purpose. An
+example of cryptographic material misuse would be if a private cryptographic
+key meant to be used for logging into a website (authentication) was instead
+used to issue a Verifiable Credential. Not checking a verification relationship
+is dangerous because some cryptographic material is protected more than other
+material, such as storage in a hardware security module in a data center versus
+as an unencrypted file on a laptop.
```

Comment:
I think a reversal of the example here is the more likely threat (plus some text on restricted use):

```suggestion
This process is used to ensure that cryptographic material, such as a private
cryptographic key, is not misused by application to an unintended purpose. An
example of cryptographic material misuse would be if a private cryptographic
key meant to be used to issue a Verifiable Credential was instead used to
log into a website (authentication). Not checking a verification relationship
is dangerous because the restriction and protection profile for some
cryptographic material could be determined by its intended use. For example,
some applications could be trusted to use cryptographic material for only
one purpose, or some cryptographic material could be more protected,
such as through storage in a hardware security module in a data center
versus as an unencrypted file on a laptop.
```

In updating this suggestion, I also noticed we need a section on proof
purpose too -- with similar text:

```
Proof Purpose Validation

When an implementation is <a href="#verify-proof">verifying a proof</a>, it is
imperative that it verify that the <a>proof purpose</a> match the intended use.

This process is used to ensure that proofs are not misused by an application
for an unintended purpose, as this is dangerous for the proof creator. An
example of misuse would be if a proof that stated its purpose was for securing
assertions in Verifiable Credentials was instead used for authentication to
log into a website. In this case, the proof creator attached proofs to any number
of Verifiable Credentials that they expected to be distributed to an unbounded
number of other parties. Any one of these parties could log into a website as
the proof creator if the website erroneously accepted such a proof as
authentication instead of its intended purpose of securing assertions.
```